scrapers/freshmaker.py

```diff
@@ -87,7 +78,10 @@ def query_api_and_update_neo4j(self):
 
                 event.conditional_connect(event.triggered_by_advisory, advisory)
 
-                for build_dict in fm_event['builds']:
+                event_builds_url = ('https://freshmaker.engineering.redhat.com/api/2/builds/'
```

Comment:
@sarah256 it seems like this API endpoint uses pagination. Could you make sure your code gets all the builds for every page? It might also help to increase the `per_page` limit so that there is less network IO involved.